packages/cli/generate_docs.sh

```diff
@@ -0,0 +1,5 @@
+set -e
+export COLUMNS=88
+yarn oclif-dev readme --multi --dir=../docs/command-line-interface
+yarn prettier --write ../docs/command-line-interface/*
+sed -i '' '/^- \[/d' ../docs/command-line-interface/*
```

Comment:
i think this line breaks the CI test on linux, [possible fix](https://stackoverflow.com/a/57766728)